composer.json

```diff
@@ -9,8 +9,8 @@
     "license": "MIT",
     "authors": [
         {
-            "name": "simivar",
-            "email": "simivar@gmail.com"
+            "name": "domenikus191",
+            "email": "domenikus191@googlemail.com"
```

Comment:
You've overwritten the original author of the library here. Please do not do that.

Reply:
Sry, my bad. I tested something and forgot that i had created a PR. Commits are reverted now